src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityMiner.java

```diff
@@ -130,20 +130,21 @@ protected ModularUI createUI(@Nonnull EntityPlayer entityPlayer) {
     }
 
     private void addDisplayText(@Nonnull List<ITextComponent> textList) {
+        int workingArea = getWorkingArea(minerLogic.getCurrentRadius());
         textList.add(new TextComponentTranslation("gregtech.machine.miner.startx", this.minerLogic.getX().get()));
         textList.add(new TextComponentTranslation("gregtech.machine.miner.starty", this.minerLogic.getY().get()));
         textList.add(new TextComponentTranslation("gregtech.machine.miner.startz", this.minerLogic.getZ().get()));
-        textList.add(new TextComponentTranslation("gregtech.machine.miner.radius", this.minerLogic.getCurrentRadius()));
+        textList.add(new TextComponentTranslation(I18n.format("gregtech.universal.tooltip.working_area", workingArea, workingArea)));
```

Comment:
This call to I18n.format() is unneeded, and will likely crash on a dedicated server.